lib/registrant.rb

```diff
@@ -0,0 +1,57 @@
+require 'pry'
+
+class Registrant
+  attr_reader :name, 
+              :age, 
+              :permit, 
+              :license_data
+
+  def initialize(name, age, permit = false)
```

Comment:
indentation in this file off here too. The def initialize's end doesn't line up...